cmd/istioctl/inject.go

```diff
@@ -105,6 +105,9 @@ kubectl get deployment -o yaml | istioctl kube-inject -f - | kubectl apply -f -
 			if versionStr == "" {
 				versionStr = version.Line()
 			}
+			if tag == "" {
+				tag = versionStr
```

Comment:
version.Line() is actually a long line with golang version, SHA, and no release version.